particle_filter:
  ros__parameters:
    num_particles: 200
    resample_threshold: 0.5
    map_yaml: maps/map.yaml
    odom_topic: /odom
    scan_topic: /scan
    publish_rate: 10.0
    beam_subsample: 10
    sigma_hit: 0.2
    z_hit: 0.8
    z_rand: 0.2
    z_max: 3.5
    alpha1: 0.02
    alpha2: 0.02
    alpha3: 0.02
    alpha4: 0.02
    map_frame: map
    robot_frame: odom
    publish_tf: false
    initialization_mode: uniform  # options: 'uniform', 'manual', 'odom'
    initial_std_xy: 0.25
    initial_std_theta: 0.25
